routers/users.py

```
@@ -0,0 +1,93 @@
+
+    token_data = {
+        "sub": user["email"],
+        "user_id": user["id"]
+    }
+    access_token = auth.create_access_token(token_data)
+
+    return Token(access_token=access_token, token_type="bearer")
+
+
+@router.get("/me", response_model=UserResponse)
+async def get_current_user_profile(current_user: dict = Depends(auth.get_current_user)):
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)